### DISCUSSION TITLE
Desperately Need an Android Version!

### DISCUSSION THREAD

**Discussion by u2n** [Motion-Project/motion#1698] [General]
With old phones already having good cameras, it would be most efficient to do motion detection right at the source, then stream only changing frames (rather than streaming all frames to a second machine for analysis).

The logo indicates some knowledge of Android (lol) so maybe not that hard to do?

EDIT 
Closed without comment a year later. I'll take that as a 'No' lol.

**Comment by Mr-Dave** [comment]
> EDIT
Closed without comment a year later. I'll take that as a 'No' lol.

Per the [poll](https://github.com/Motion-Project/motion/discussions/1794) discussion, all enhancements to Motion have been closed.  Additionally, I wouldn't have any interest in writing a new application for Android.

**Comment by u2n** [comment]
Thanks, @Mr-Dave, that clears it up.

In the year since opening this, I've learned to note graphics throughput demands. Detection would, in most cases, have to operate at low-resolution, then trigger hi-res for recording or streaming. Not as simple as I thought.

Many thanks for all your work on this.
